packages/lib/apps/mergeUserAndTeamAppCredentials.ts

```diff
@@ -0,0 +1,23 @@
+import type { TeamQuery } from "@calcom/ee/teams/teams.repository";
+
+import getTeamAppCredentials from "./getTeamAppCredentials";
+import type { Credentials } from "./getTeamAppCredentials";
+
+const mergeUserAndTeamAppCredentials = (
+  userTeams: TeamQuery[],
+  userCredentialIds: Credentials,
+  includeTeamInstalledApps?: boolean
+) => {
+  const teamAppCredentials = getTeamAppCredentials(userTeams);
+  let credentials: Credentials = userCredentialIds;
+
+  if (!includeTeamInstalledApps) {
+    credentials = teamAppCredentials;
+  } else {
+    credentials = userCredentialIds.concat(teamAppCredentials);
+  }
+
+  return credentials;
```

Comment:
```suggestion
  return !includeTeamInstalledApps ? teamAppCredentials : userCredentialIds.concat(teamAppCredentials);
```

Reply:
Assigment here is not adding clarity. The function name already handles the expectation of what is returning,

Reply:
Agreed, will remove the assignment. But instead of using a ternary like you mentioned above I did an early return, reason being if in the future if someone wants to add another condition, its very easy to make a mess out of the ternary operator. What I did now is as follows:

```
if (!includeTeamInstalledApps) {
    return teamAppCredentials;
  }

return userCredentialIds.concat(teamAppCredentials);
```